sensu/files/plugins/graphite-data.rb

```diff
@@ -43,16 +43,16 @@ class CheckGraphiteData < Sensu::Plugin::Check::CLI
     :required => false
 
   option :warning,
-    :description => 'Generate warning if given value is above received value',
+    :description => 'Generate warning if given value is above received value. Or comma seperated range.',
     :short => '-w VALUE',
     :long => '--warn VALUE',
-    :proc => proc{|arg| arg.to_f }
```

Comment:
Does that split to an array and .to_f the elements? If so that's good ruby magic!